src/main/java/seedu/address/ui/ResultDisplay.java

```diff
@@ -36,6 +37,18 @@ public ResultDisplay() {
     private void handleNewResultAvailableEvent(NewResultAvailableEvent event) {
         logger.info(LogsCenter.getEventHandlingLogMessage(event));
         Platform.runLater(() -> displayed.setValue(event.message));
+
+        if(event.isError) setErrorStyle();
```

Comment:
Checkstyle violation reported correctly. Did you configure your checkstyle correctly?